### DISCUSSION TITLE
town rank

### DISCUSSION THREAD

**Discussion by Erissos** [TownyAdvanced/Towny#6722] [Q&A] [answered]
![image](https://github.com/TownyAdvanced/Towny/assets/88965431/b63226dd-527e-4726-90cd-f0f8fb61ddd3)

There are ranks in the places I marked in red in the picture. I want to change rank views.

**Comment by LlmDl** [comment]
I don't understand your question, or how you're displaying the ranks, I'm assuming you want to change how the formatting is. 

https://github.com/TownyAdvanced/Towny#a-special-message-from-llmdl-precedes-the-normal-readme-

**Comment by Erissos** [comment]
" in towny i cant change the language of the ranks what should i do "

**Comment by LlmDl** [comment, marked as answer]
You can change the rank names in the townyperms.yml.

Note: as per the townyperms.yml comments there are some ranks you cannot change.
